lib/modules/manager/helmfile/schema.ts

```diff
@@ -11,16 +11,21 @@ export type HelmRepository = z.infer<typeof HelmRepository>;
 export const HelmRelease = z.object({
   name: z.string(),
   chart: z.string(),
-  version: z.string(),
+  version: z
+    .string()
+    .or(z.number())
```

Comment:
parser issue? is this valid for helm?

Reply:
This is to cover cases where `version: 12.2` or similar are used. We have such example in the fixtures.